lib/canvas_editor/events.js

```diff
@@ -70,23 +70,27 @@ function addPointerListeners(canvasElement, drawArea, JavaEditorArea) {
 function addKeyboardListeners(canvasElement, editorArea, JavaEditorArea) {
   const isMac =
     typeof navigator !== 'undefined' && navigator.platform === 'MacIntel';
+  const isMenuKey = (ev) => (isMac && ev.metaKey) || (!isMac && ev.ctrlKey);
 
   function fireKeyEvent(what, ev) {
     const key = getKeyFromEvent(ev, JavaEditorArea);
     if (key === null) return;
-    ev.stopPropagation();
-    ev.preventDefault();
```

Comment:
Removed so browser can propagate copy-paste events.
To interact with clipboard, browser need proper user interaction